python/lsst/daf/butler/datastores/fileLikeDatastore.py

```diff
@@ -246,21 +247,22 @@ def getStoredItemInfo(self, ref):
                               checksum=record["checksum"],
                               file_size=record["file_size"])
 
-    def getStoredItemInfoForPath(self, pathInStore):
-        # Docstring inherited from GenericBaseDatastore
-        records = list(self.registry.fetchOpaqueData(self._tableName, path=pathInStore))
-        if len(records) == 0:
-            return records
+    def _registered_refs_per_artifact(self, pathInStore):
+        """Return all dataset refs associated with the supplied path.
 
-        for r in records:
-            print(r["path"])
+        Parameters
+        ----------
+        pathInStore : `str`
+            Path of interest in the data store.
 
-        return [StoredFileInfo(formatter=r["formatter"],
-                               path=r["path"],
-                               storageClass=self.storageClassFactory.getStorageClass(r["storage_class"]),
-                               checksum=r["checksum"],
-                               file_size=r["file_size"])
-                for r in records]
+        Returns
+        -------
+        refs : `set` of `int`
```

Comment:
I'd call this something other than "refs" (same with the method name).  Maybe "ids"?